api/README.md

```
@@ -38,6 +38,11 @@ Copy the generated secret into `JWT_SECRET_KEY` in `api/.env`. See [`../docs/ENV
 bin/rails db:prepare
 ```
 
+#### Optional: Reset the database
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)